tests/test_errors.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from search.errors import NAMESPACE_MAP
+from search.xml import parse_xml_from_bytes
+
+
+@pytest.mark.parametrize('status_code, name', [
+    (400, 'Bad Request'),
+    (401, 'Unauthorized'),
+    (403, 'Forbidden'),
+    (404, 'Not Found'),
+    (405, 'Method Not Allowed'),
+    (406, 'Not Acceptable'),
+    (408, 'Request Timeout'),
+    (409, 'Conflict'),
+    (410, 'Gone'),
+    (411, 'Length Required'),
+    (412, 'Precondition Failed'),
```

Comment:
tests are currently failing for error code 412. I've opened an issue [here](https://github.com/pallets/flask/issues/3093)

Reply:
This appears to be by design. @thewilkybarkid should I just comment this out or remove 412 from the test?

Reply:
Technically we won't need this code for this service (since it's read only), but we will want it for future writeable apps.

Reply:
So should I comment it out or remove it from the test?

Reply:
I've commented this out for now and left a small explanation, referencing the flask issue I raised.